test/testHTMLFromMarkdown.js

```diff
@@ -98,33 +96,28 @@ const assertMd = async (md, html) => {
     },
   );
 
-  eq(
```

Comment:
@ramboz the tests never actually failed, because the `eq` only tests for equality. but doesn't throw if not equal :-)